workers/src/main/kotlin/br/com/messore/tech/turtleracing/workers/RunnerWorker.kt

```diff
@@ -1,30 +1,32 @@
 package br.com.messore.tech.turtleracing.workers
 
 import android.content.Context
+import android.util.Log
+import androidx.hilt.work.HiltWorker
 import androidx.work.CoroutineWorker
 import androidx.work.Data
 import androidx.work.OneTimeWorkRequestBuilder
 import androidx.work.WorkManager
 import androidx.work.WorkRequest
 import androidx.work.WorkerParameters
+import br.com.messore.tech.turtleracing.domain.usecase.token.PrinterUseCase
 import dagger.assisted.Assisted
 import dagger.assisted.AssistedFactory
 import dagger.assisted.AssistedInject
 import java.time.Duration
+import javax.inject.Inject
 
+@HiltWorker
 class RunnerWorker @AssistedInject constructor(
     @Assisted context: Context,
-    @Assisted workerParams: WorkerParameters
+    @Assisted workerParams: WorkerParameters,
+    val useCase: PrinterUseCase
```

Comment:
Poderia ser private?
```suggestion
    private val useCase: PrinterUseCase
```

Reply:
Imagino que esse usecase estava somente para testes, acha que rolaria removê-lo?

Reply:
Exatamente deixei apenas para termos um exemplo de utilização, porem posso remover para ficar correto desde o começo.